rest/route/project_test.go

```diff
@@ -776,3 +776,20 @@ func TestDeleteProject(t *testing.T) {
 	resp = pdh.Run(ctx)
 	assert.Equal(t, http.StatusBadRequest, resp.Status())
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// Tests for PUT /rest/v2/projects/value
+
+func (s *ProjectPutSuite) TestNotSureIfThisRight() {
```

Comment:
i'm assuming the name of this test is a mistake haha

Reply:
oh no